gnosis/cowswap
# Fix Affiliate error banner showing up

# Summary

Closes #1749

This fixes an issue where an invalid referral banner was displayed after refreshing the page where no referral query parameter was present. Cause: An invalid referral was set in the address bar and persisted in storage, later on through navigation the referral query parameter is removed but the error banner is still showing up because the referral object was still present in storage.

  # To Test

1. Connect the wallet
2. Go to /profile and copy the referral url
3. Paste the referral url into the address bar and remove the last character from the referral address, an error banner saying the referral is invalid should appear
4. Close the error banner and go to another url, like /about
5. Refresh the page

ER: The error banner shouldn't appear again
